src/com/tinyinswing/demo/Demo.java

```diff
@@ -0,0 +1,79 @@
+package com.tinyinswing.demo;
+
+import cloud.tiny.tinymceforswing.TinyMCE;
+import cloud.tiny.tinymceforswing.api.config.Config;
+import cloud.tiny.tinymceforswing.api.config.wrappers.ActionApi;
+import cloud.tiny.tinymceforswing.api.events.TinyListener;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.util.concurrent.ExecutionException;
+
+public class Demo extends JFrame {
+
+  public Demo() {
+    try {
+      init(this);
+    } catch (ExecutionException e) {
+      e.printStackTrace();
```

Comment:
I would just put `throws ExecutionException, InterruptedException` on the call chain.